bin/keycloak-httpd-client-install

```diff
@@ -829,24 +833,29 @@ def build_template_params(options):
 
 def build_mellon_httpd_config_file(options):
     template_params = build_template_params(options)
-    template = template_env.get_template(MELLON_HTTPD_CONFIG_TEMPLATE)
+    if options.module == 'mod_auth_mellon':
+        template = template_env.get_template(MELLON_HTTPD_CONFIG_TEMPLATE)
+    elif options.module == 'mod_auth_openidc':
+        template = template_env.get_template(OIDC_MELLON_HTTPD_CONFIG_TEMPLATE)
     return template.render(template_params)
 
-
 def build_mellon_sp_metadata_file(options):
```

Comment:
A function whose name includes "mellon" should not be producing data applicable to both mellon and oidc.